src/content/tooling/wrangler/commands.md

```diff
@@ -138,6 +138,17 @@ If you would like to be able to publish your code to multiple places, please see
   If you have feedback about `wrangler dev` or general questions, we will respond [here](https://github.com/cloudflare/wrangler/issues/1047).
 
 
+### tail
+
+#### Dependencies
+
+  Wrangler tail uses cloudflared under the hood. If you are already using cloudflared, be sure you have installed the latest version. Otherwise, follow the [getting started guide](https://developers.cloudflare.com/argo-tunnel/quickstart/) for Argo Tunnel.
+
+#### Usage
+
+  `wrangler tail` will register a tailing session for your Worker, and start a server on `localhost` with a [tunnel](https://developers.cloudflare.com/argo-tunnel/quickstart/) that listens for incoming log requests from your Worker. After starting `wrangler tail` in a directory with a project, you will receive a live feed of console and exception logs for each request your Worker receives.
+
+  Like all Wrangler commands, you should run `wrangler tail` from your Worker's root directory. You can configure the port for your local log server by passing the `--port` flag. An additional port is used for serving metrics information for the tunnel; this can be configured by passing the `--metrics-port` flag.
```

Comment:
i haven't used the `metrics-port` flag yet, but maybe it would be useful to show example output here?

Reply:
i think actually i should link to the argo tunnel docs about this endpoint, as it is orthogonal to wrangler tail but necessary to add to config since it demands  its own port.